porcupine/plugins/comment_selected_lines.py

```diff
@@ -14,6 +14,15 @@
 from porcupine.plugins import rightclick_menu
 
 
+def already_commented(linetext: str, comment_prefix: str) -> Optional[bool]:
+    # Ignore '# blah' comments because they are likely written by hand
```

Comment:
If I'm trying to uncomment something, does it matter how it was written?

Reply:
> \# Ignore '# blah' comments because they are likely written by hand

Or `#    blah` comments, formatted with black

Reply:
@Moosems @rdbende I completely agree. However, @Akuli wants the functionality like this. My first version of this code always did `# foo`, no matter the indentation. However, seems like Akuli wants to separate manually added comments and comments made with keybindings.

Reply:
@rdbende I think formatting `#        blah` with black actually does nothing. It seems to be left like that!

Reply:
```console
$ cat t.py
def foo():
#    bar = 8
#    bar += 2
#    assert bar == 10
    ...
$ black t.py
reformatted t.py

All done! ✨ 🍰 ✨
1 file reformatted.

$ cat t.py
def foo():
    #    bar = 8
    #    bar += 2
    #    assert bar == 10
    ...
```